bucket/dokan.json

```diff
@@ -0,0 +1,36 @@
+{
+    "version": "1.3.1.1000",
+    "description": "User mode file system for Windows",
+    "homepage": "https://dokan-dev.github.io/",
+    "license": "LGPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/dokan-dev/dokany/releases/download/v1.3.1.1000/Dokan_x64.msi",
+            "hash": "1b89e2f3cf90902726a5d03a9694596a3a247d6908b74ab493469b259d8f8462"
+        },
+        "32bit": {
+            "url": "https://github.com/dokan-dev/dokany/releases/download/v1.3.1.1000/Dokan_x86.msi",
+            "hash": "b296e7bef4c6ca45f10b6c1b444ff2e38dec40c79753ca24f26f19c1a6aaaf0b"
+        }
+    },
+    "bin": [
+        "dokanctl.exe",
+    ],
+    "checkver": {
+        "github": "https://github.com/dokan-dev/dokany"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/dokan-dev/dokany/releases/download/v$version/Dokan_x64.msi"
+            },
+            "32bit": {
+                "url": "https://github.com/dokan-dev/dokany/releases/download/v$version/Dokan_x86.msi"
+            }
+        },
+        "extract_dir": "Dokan\\DokanLibrary-$matchHead"
+    },
+    "suggest": {
```

Comment:
@Ash258 why does your comment in old PR remove suggestion for vcredist2019? installation fails without it.